tools/c7n_azure/tests/test_storage.py

```diff
@@ -19,6 +19,22 @@
 class StorageTest(BaseTest):
     def setUp(self):
         super(StorageTest, self).setUp()
+        self.cleanup_policy = self.load_policy({
```

Comment:
I would prefer if you move this `cleanup_policy` into the teardown.

Reply:
self.cleanup_policy is more important to have in the beginning of two particular tests, not at the end of all of them